azure/android-pipeline.yml

```diff
@@ -1,7 +1,7 @@
 trigger:
```

Comment:
Presumably these files can be deleted?

Reply:
Yes, I just wanted to keep both until it's merged.